modules/cert-manager-configure-cloud-credentials-gcp-sts.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-authenticate-gcp.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-configure-cloud-credentials-gcp-sts_{context}"]
+= Configuring cloud credentials for the {cert-manager-operator} with GCP Workload Identity
+
+To configure the cloud credentials for the {cert-manager-operator} on a GCP Workload Identity cluster, you must generate the cloud credentials manually by using the `ccoctl` binary, and apply it on the cluster.
+
+.Prerequisites
+
+* You must extract and prepare the `ccoctl` binary.
+* The {cert-manager-operator} 1.11.1 or later is installed.
+* You have configured an {product-title} cluster with GCP Workload Identity by using the Cloud Credential Operator in a manual mode.
+
+.Procedure
+
+. Create a directory to store a `CredentialsRequest` resource YAML file by running the following command:
++
+[source,terminal]
+----
+$ mkdir credentials-request
+----
+
+. Create a `CredentialsRequest` resource YAML file under the `credentials-request` directory, such as, `sample-credential-request.yaml` by applying the following yaml:
++
+[source,yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: cert-manager
+  namespace: openshift-cloud-credential-operator
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: GCPProviderSpec
+    predefinedRoles:
+    - roles/dns.admin
+  secretRef:
+    name: gcp-credentials
+    namespace: cert-manager
+  serviceAccountNames:
+  - cert-manager
+----
++
+[NOTE]
+====
+The `dns.admin` role provides admin privileges to the service account for managing Google Cloud DNS resources. To ensure cert-manager runs with the service account that has the least privilege, you must create a custom role with the following permissions:
+
+* `dns.resourceRecordSets.*`
+* `dns.changes.*`
+* `dns.managedZones.list`
+====
+
+. Use the `ccoctl` tool to process `CredentialsRequest` objects by running the following command:
```

Comment:
Future improvement suggestion: add an example of this command with all of the user-replaceable values substituted with example values.